pkgs/data/fonts/alice/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, stdenv, fetchzip }:
+
+stdenv.mkDerivation (attrs: {
+  pname = "Alice";
+  version = "2.003";
+
+  outputs = [ "out" "woff2" ];
+
+  src = fetchzip {
```

Comment:
`fetchFromGitHub` doesn't work?

Reply:
We could fetch the `v2.003` tag, but the source is slightly larger (6 MiB) than the release tarball (288 KiB) so I went with the latter.